storage.py

```diff
@@ -17,6 +17,14 @@ def remove(self):
 
     def set(self):
         pass
-    
-    def add(self):
-        pass
\ No newline at end of file
+
+    def add(self, key, value):   #Катя
+        key = key
+        value = value
```

Comment:
It is unnecessary to create local variables _key_, _value_ in function as they are already passed into as arguments (these lines can be removed).

Reply:
fixed it :)